modules/test/playwright/tests/frontend-data-set-views-web/dataSetsSample.spec.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import {pageEditorPagesTest} from '../layout-content-page-editor-web/fixtures/pageEditorPagesTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPS-164563': true,
+	}),
+	loginTest(),
+	pageEditorPagesTest
+);
+
+test('Add the frontend data set sample widget', async ({apiHelpers, page}) => {
+	let site: Site;
+
+	await test.step('Create a content site and the frontend data set sample widget', async () => {
+		site = await apiHelpers.headlessSite.createSite('Test Site');
+
+		await apiHelpers.headlessDelivery.createSitePage(
+			site.id,
+			getRandomString(),
+			{
+				pageElement: {
+					definition: {
+						widgetInstance: {
+							widgetName:
+								'com_liferay_frontend_data_set_sample_web_internal_portlet_FDSSamplePortlet',
+						},
+					},
+					id: getRandomString(),
+					type: 'Widget',
+				},
+			}
+		);
+	});
+
+	await test.step('Assert that the filter client extension is added', async () => {
+		await page.getByRole('tablist').getByText('Customized').click();
```

Comment:
Don't you need to navigate to the created page first?

Reply:
How to do that **exactly**? @markocikos 

I saw you used 
```js
await page.goto(`${liferayConfig.environment.baseUrl}/web${site.friendlyUrlPath}${layout.friendlyUrlPath}`);
```

After adapting my code to match yours (`layout` + `getPageDefinition` setup)

But for me, in Playwright UI, that page is empty - looking like this
<img width="857" alt="image" src="https://github.com/liferay-frontend/liferay-portal/assets/24564752/8b4b55fe-be0b-4a0e-90e0-ccb4da0ad9aa">

Reply:
@markocikos pushed https://github.com/liferay-frontend/liferay-portal/pull/4047/commits/b142f6a03d5b33fb450ec99a55d9172ab6ec1f5c

Reply:
this works with the sample + cx deployed and sample on the page, doesn't work with fresh DXP instance. 
Is that intended @markocikos I though that the test should do it all on its own so its compatible with CI

Reply:
@kresimir-coko it doesn't work for me locally
![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/f03bcfea-3766-428d-8668-0669645948b2)

Reply:
Your usage of `page.x` does not look right:

![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/0305f5fa-490d-4219-9daa-e6cd28ab4fb8)

Reply:
I commented out site deletion in `isolatedSiteTest.ts` to see if the portlet deployed, and it did:
![image](https://github.com/liferay-frontend/liferay-portal/assets/5435511/b48c6c98-9971-4866-ad88-da6713db24ec)

It's your tests that are not right.